guides/common/modules/proc_configuring-selinux-to-permit-access-to-repos-on-custom-ports.adoc

```diff
@@ -0,0 +1,27 @@
+[id="configuring-selinux-to-ensure-access-on-custom-ports_{context}"]
+= Configuring SELinux to Permit Access to Repositories on Custom Ports
+
+ifdef::satellite,foreman-el,katello[]
+SELinux permits access of {ProjectNameX} to repositories only on specific ports.
```

Comment:
```suggestion
SELinux permits access of {Project} to repositories only on specific ports.
```
Do we really need to show the specific version?

Reply:
No, good point, will fix